docs/chapter_preface/suggestions.md

```diff
@@ -172,7 +172,14 @@
 === "Ruby"
 
     ```ruby title=""
+    ### 标题注释，用于标注函数、类、测试样例等 ###
 
+    # 内容注释，用于详解代码
+    
+    =begin
```

Comment:
Hi, @khoaxuantu. Which one do you like for multi-line comments?

```ruby
# 多行
# 注释
```

Reply:
I think using `#` is preferable. The `=begn ... =end` block cannot be indent.

```
# Comment
# Something

  # An indentation comment
  # Something
```

Reply:
> OK, I will fix it. I have read ur codes and found that `=begin ... =end` has been used.
> 
> I'm new in github. This is the first open source project I've committed to. I may make mistakes and hope you can help me.

We use `=begin ... =end` as the header file comment to indicate the file, the created time, and the author. For the other comments, `#` is enough to use.